provider/pkg/provider/helm_release.go

```diff
@@ -1758,3 +1683,23 @@ func getTimeoutOrDefault(timeout int) time.Duration {
 	}
 	return time.Duration(timeout) * time.Second
 }
+
```

Comment:
Can we add a brief godoc here to describe why no schema (ie. empty return in the implemented functions) is sufficient in our use case?